src/components/sidebar/cart/Cart.test.js

```diff
@@ -35,12 +35,23 @@ describe('Cart', () => {
     expect(wrapper.get(0)).toMatchSnapshot();
   });
 
+  it('opens the modal when clicked', () => {
+    mockUseSelector([{ id: '123' }, { id: '123' }]);
+
+    const classes = mockStyles(styles);
+    const wrapper = shallow(<UnstyledCart classes={classes} />);
+    wrapper.find(Button).first().simulate('click');
```

Comment:
After taking a look at which tests were failing, it seems like `Cart` complains about rendering after this button is clicked (same with the test below), because `sections` gets cleared as intended and then becomes `undefined` for some reason? I don't think I understand the selector mock well enough to know what's going on, pinging @Joonpark13 for help

Reply:
Found a bug in `mockUseSelector`, good catch. I'll slack you both.